diff --git a/node_modules/child-process-promise/node_modules/cross-spawn/lib/parse.js b/node_modules/child-process-promise/node_modules/cross-spawn/lib/parse.js
index 77cbb83..99cf774 100644
--- a/node_modules/child-process-promise/node_modules/cross-spawn/lib/parse.js
+++ b/node_modules/child-process-promise/node_modules/cross-spawn/lib/parse.js
@@ -52,12 +52,13 @@ function escapeArg(arg, quote) {
     } else {
         // Sequence of backslashes followed by a double quote:
         // double up all the backslashes and escape the double quote
-        arg = arg.replace(/(\\*)"/g, '$1$1\\"');
+        // Altered to disable JS backtracking to avoid hanging on specially crafted input (CVE-2024-21538)
+        arg = arg.replace(/(?=(\\+?)?)\1"/g, '$1$1\\"');
 
         // Sequence of backslashes followed by the end of the string
         // (which will become a double quote later):
         // double up all the backslashes
-        arg = arg.replace(/(\\*)$/, '$1$1');
+        arg = arg.replace(/(?=(\\+?)?)\1$/, '$1$1');
 
         // All other backslashes occur literally
 
