--- a/google/firestore/v1/write.proto
+++ b/google/firestore/v1/write.proto
@@ -38,6 +38,11 @@
     // A document name to delete. In the format:
     // `projects/{project_id}/databases/{database_id}/documents/{document_path}`.
     string delete = 2;
 
+    // The name of a document on which to verify the `current_document`
+    // precondition.
+    // This only requires read access to the document.
+    string verify = 5;
+
     // Applies a transformation to a document.
     DocumentTransform transform = 6;
   }